Elevel/Application/Features/TopicCommands/UpdateTopicCommand.cs

```diff
@@ -39,12 +42,20 @@ public Handler(IApplicationDbContext context, IMapper mapper)
 
             public async Task<Response> Handle(Request request, CancellationToken cancellationToken)
             {
+                var validator = new Validator();
```

Comment:
validator doesn't need here, it will be executed automatically by mediatr pipeline